packages/cadl-ranch-specs/http/server/versions/not-versioned/main.tsp

```diff
@@ -0,0 +1,38 @@
+import "@typespec/http";
+import "@azure-tools/cadl-ranch-expect";
+
+using TypeSpec.Http;
+
+/**
+ * Illustrates not-versioned server.
+ */
+@scenarioService("/server/versions/notVersioned")
```

Comment:
nit, kebab-casing
https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#http-url-casing

same for `@route`